AHK plugin/Sdl.Community.AhkPlugin/ViewModels/ImportScriptPageViewModel.cs

```diff
@@ -17,31 +20,32 @@ public class ImportScriptPageViewModel:ViewModelBase
 	    private ICommand _backCommand;
 	    private ICommand _dragEnterCommand;
 	    private ICommand _removeFileCommand;
+	    private ICommand _addFilesCommand;
 	    private ICommand _addToMasterCommand;
 	    private ICommand _changeScriptStateCommand;
 	    private ICommand _selectAllCommand;
-	    private readonly DbContext _dbContext;
+	    private readonly IDialogService _dialogService;
+		private readonly DbContext _dbContext;
 		private string _gridVisibility;
 	    private string _message;
 	    private string _messageVisibility;
 	    private bool _selectAll;
 		private ObservableCollection<KeyValuePair<string,Script>> _scriptsCollection = new ObservableCollection<KeyValuePair<string, Script>>();
 		private ObservableCollection<ImportScriptItemTemplate> _filesNameCollection = new ObservableCollection<ImportScriptItemTemplate>();
+		private const string FilesFilter = "AHK Scripts(*.ahk) | *.ahk";
 
 		public static readonly Log Log = Log.Instance;
 
 		public ImportScriptPageViewModel(MainWindowViewModel mainWindowViewModel)
 		{
 			_mainWindowViewModel = mainWindowViewModel;
+			_dialogService = new FilesDialogService();
```

Comment:
Should we consider instantiating the service using Dependency injection and take advantage of the interface created?